assets/js/base/context/hooks/test/use-store-notices-with-snackbar.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { render, act } from '@testing-library/react';
+import { StoreNoticesWithSnackbarProvider } from '@woocommerce/base-context/providers';
```

Comment:
I have one small recommendation, namely to rename to `StoreSnackbarNoticesProvider`. You already have `SnackbarNoticesContainer`, so it would be consistent to refer to them as `SnackbarNotices`. I'm thinking if another type of notification appears we could keep the pattern

- NoticesProvider
- SnackbarNotices
- XNotices